tiledb/sm/c_api/tiledb.h

```diff
@@ -901,14 +901,34 @@ TILEDB_EXPORT int32_t tiledb_array_schema_has_attribute(
  *
  * @param ctx The TileDB context.
  * @param array_schema The array schema.
- * @param out The output.
+ * @param out The output file.
  * @return `TILEDB_OK` for success and `TILEDB_ERR` for error.
  */
 TILEDB_EXPORT int32_t tiledb_array_schema_dump(
     tiledb_ctx_t* ctx,
     const tiledb_array_schema_t* array_schema,
     FILE* out) TILEDB_NOEXCEPT;
 
+/**
+ * Dumps the array schema in ASCII format in the selected string output.
+ *
```

Comment:
Documentation needs a paragraph that states that the output string handle must be freed after use. The example should also illustrate this.